ukol_03/sifra.py

```diff
@@ -0,0 +1,14 @@
+plain_text = input("Please, insert your message to encrypt: ")
+shift = int(input("Please, insert shift: "))
+result = ""
+
+for character in plain_text:
+    if character == " ": 
+        result = result + " "
+    else: 
+        x = ord(character)
+        result += chr(x + shift)
```

Comment:
Chápem, že si to chcela čo najviac univerzálne... ale nebolo by to lepšie limitovať len na písmenká anglickej abeced? Taký hint: Pozri si ako za sebou idú tieto čísla v ASCII tabuľke.

Nakoľko ak tam aktuálne zadám určitú postupnosť znakov a posun (napríklad záporný) tak sa pokúsi previesť naše znaky na systémové akcie... :smile_cat: čo nie je práve správne. Preto by to bolo lepšie trocha obmedziť.